cohortextractor/main.py

```diff
@@ -1,18 +1,33 @@
 import csv
 import importlib.util
 import inspect
+import shutil
 import sys
 from contextlib import contextmanager
+from pathlib import Path
 
-from .backends import BACKENDS
-from .query_utils import get_column_definitions
+from cohortextractor.backends import BACKENDS
```

Comment:
What's the reason for changing from relative to absolute imports here?

Reply:
At one point the tests refused to run with the relative imports (even using `just` which should be setting the PYTHONPATH).  I'll see if they'll work if I change them back now

Reply:
I don't mind at all -- just wanted to improve my understanding for this stuff.